.travis.yml

```diff
@@ -1,23 +1,27 @@
 # see http://about.travis-ci.org/docs/user/languages/php/ for more hints
 language: php
+
 php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
```

Comment:
@TomasVotruba Ok for that but can you add `nightly` tag also?

With that, we will already prepared for next release after PHP 7 stable. :+1: